Source/Lib/Codec/EbEntropyCoding.c

```diff
@@ -5412,14 +5434,17 @@ static void CodeVPS(
 	{
 		// Supports Main Profile Level 5 (Max Picture resolution 4Kx2K, up to 6 Hierarchy Levels)
 		// Max Dec Pic Buffering [i]
-		WriteUvlc(
-			bitstreamPtr,
+	    WriteUvlc(
+		    bitstreamPtr,
 			scsPtr->maxDpbSize - 1/*4*/);//(1 << (scsPtr->maxTemporalLayers-syntaxItr)));
 
+		// Max Num Reorder pics - indicates the maximum number of pictures that can precede any picture in
```

Comment:
pls keep comment same as in spec: vps_max_num_reorder_pics to make it easier to search in the code.
Also keep the same style as other comments.